### ISSUE TITLE
Special cases for films with french subtitles

### ISSUE THREAD

**Issue by sinopsysHK** [platelminto/parse-torrent-name#58]
Many french films names do have VOSTFR standing for Original audio soundtrack with French Subtitles.

Hence film like "The Good German (2006).**VOSTFR**.720p.WEBDL.h264.aac.mkv" should hav subtittle = "fr"
instead of taking it as release group
```json
{
'year': '2006',
'resolution': '720p',
'title': 'The Good German',
'release_title': 'The Good German (2006).VOSTFR.720p.WEBDL.h264.aac',
'videocodec': 'H.264',
'audiocodec': 'AAC',
'source': None,
'releasegroup': 'VOSTFR',
'container': None
}
```

Same with films with **subfrench** in the name such as "The.Walking.Dead.S06E07.SUBFRENCH.HDTV.x264-AMB3R.mkv" which is here taking subfrench in excess.
```json
{
'season': 6, 
'episode': 7, 
'title': 'The Walking Dead', 
'excess': 'SUBFRENCH', 
'release_title': 'The.Walking.Dead.S06E07.SUBFRENCH.HDTV.x264-AMB3R.mkv', 
'videocodec': 'H.264', 
'audiocodec': None, 
'source': None, 
'releasegroup': 'AMB3R', 
'container': 'MKV'
}
```

**Comment by platelminto** [comment]
Hey, thanks for opening the issue! Should get this fixed over the next couple days.

Slightly confused though: in the first example you give, it shows "VOSTR", not "VOSTFR" - are they both for French subs? A quick look online showed many results for the latter, but nothing for just "VOSTR".

**Comment by sinopsysHK** [comment]
Sorry this is a typo obviously VOSTFR is te correct one!

**Comment by platelminto** [comment]
Fixed, and available on PyPI! (version 2.8)
